python/tests/plugins/lsp_server/providers/test_goto_definition_provider.py

```diff
@@ -110,23 +110,22 @@ async def test_handles_goto_definition_request_when_definition_in_different_docu
         new_test_document_content = f"{TEST_DOCUMENT_CONTENT}{YAML_DOCUMENT_SEPARATOR}{TEST_SERVICE_THREE.to_yaml()}"
         await self.write_document(TEST_DOCUMENT_NAME, new_test_document_content)
 
-        target_location = self.get_definition_location_of_name(new_test_document_content, TEST_SCHEMA_C.name)
         res: GotoDefinitionResponse = await self.goto_definition(
-            TEST_DOCUMENT_NAME, line=target_location.range.start.line, character=target_location.range.start.character
+            TEST_DOCUMENT_NAME, line=61, character=17
```

Comment:
Hard-coding these values doesn't seem ideal since a change to the test document content (or more likely, a change to any of the definitions that make up the test document content) could change the specific `line` and `character` causing this test to fail.

Reply:
I agree, but the logic I would have to implement to dynamically determine those lines would resemble the logic it's intended to test.